### DISCUSSION TITLE
Dendron export node with details.

### DISCUSSION THREAD

**Discussion by v3ss0n** [dendronhq/dendron#3867] [General]
I have trouble exporting dendron, i would like to export Dendron , complete with note ref. I want to export as i see in dendron Preview. There is no feature to export individual notes.
What is the best way ?  Even if i can export single note , with included inline note references to html or pdf or GFM compatible markdown with included inline note references is fine.

**Comment by ScriptAutomate** [comment]
What should help in exporting a single note to GFM-compatible markdown, and will include your note refs, would be the Markdown Export Pod v2. Here is a quickstart for the Markdown export pod: https://wiki.dendron.so/notes/j5wgTcRHQZomzSHsg41O8/

The quickstart shows how to configure it so it will copy the export to clipboard, but you can change the configuration destination to be a filepath instead of clipboard, too.

More info related to exporting:

- Markdown Export Pod: https://wiki.dendron.so/notes/Un0n1ql7LfvMtmA9JEi4n/
- Export config common properties: https://wiki.dendron.so/notes/yTnBABnepuIcSJ5zxfwR4/
